packages/fuselage-ui-kit/src/hooks/useUiKitState.ts

```diff
@@ -45,7 +58,15 @@ export const useUiKitState: <TElement extends UiKit.ActionableElement>(
 
   const initialValue =
     (hasInitialValue(rest) && rest.initialValue) ||
+    (hasInitialTime(rest) &&
+      new Date(rest.initialTime).toLocaleTimeString(undefined, {
+        hour: 'numeric',
+        minute: 'numeric',
+      })) ||
```

Comment:
I think we made a mistake on `TimePickerElement` by using `number` instead of `${`${'0' | '1'}${'0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'}` | `2${'0' | '1' | '2' | '3'}`}:${'0' | '1' | '2' | '3' | '4' | '5'}${'0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'}`. Let's revisit the types ASAP and change it here.